src/tags/templates/tags/snippets/related-xp-item.html

```diff
@@ -6,7 +6,7 @@
             <p></p>
             <ul>
                 {% for quest in quests %}
-                    <li><a href="{{ quest.get_absolute_url }}">{{ quest }}</a></li>
+                    <li><a href="{{ quest.get_absolute_url }}">{{ quest }} ({{ quest.xp }} XP)</a></li>
```

Comment:
I don't think this is correct.   Are these quests or submissions?  Seems like they should be submissions. Otherwise this won't take into account custom xp for a submission, will it?  Also, how does this account for multiple completions of a repeatable quest?